src/state_point.F90

```diff
@@ -866,14 +790,12 @@ subroutine load_state_point()
       ! Read in sum and sum squared
       if (int_array(1) == 1) then
         TALLY_RESULTS: do i = 1, n_tallies
-
           ! Set pointer to tally
           tally => tallies(i)
-          curr_key = key_array(id_array(i))
 
           ! Read sum and sum_sq for each bin
           tally_group = open_group(tallies_group, "tally " // &
-               trim(to_str(curr_key)))
```

Comment:
Of course it is in reality much more complicated than this, if a user changes a tally in "tallies.xml" (*e.g.*, removes/adds a score) and tries to load from a statepoint no error message is reported.